react/components/SearchResultContainer.js

```diff
@@ -169,14 +169,25 @@ export default class SearchResultContainer extends Component {
           }
         )
 
+        // backwards compat
```

Comment:
Avoiding abbreviations will avoid confusion in the future. I'd suggest using the full word `compatibility`